src/common/ErrorMessage.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
why to disable this? :)

Reply:
eslint is is complainging about `'`inside the text, do I need to enable it at again the end of the file?